File: app/src/main/java/de/rafaelmiranda/memory/GameApplication.java
package de.rafaelmiranda.memory;

import android.app.Application;

import de.rafaelmiranda.memory.utils.FontLoader;

public class GameApplication extends Application {

	@Override
	public void onCreate() {
		super.onCreate();
		FontLoader.loadFonts(this);
	}
}

File: app/src/main/java/de/rafaelmiranda/memory/GameApplication.kt
package de.rafaelmiranda.memory

import android.app.Application
import com.google.firebase.firestore.FirebaseFirestore
import com.google.firebase.firestore.FirebaseFirestoreSettings


class GameApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        val firestore = FirebaseFirestore.getInstance()
        val settings = FirebaseFirestoreSettings.Builder()
                .setTimestampsInSnapshotsEnabled(true)
                .build()
        firestore.firestoreSettings = settings
    }
}
